test/quantized_ops/test_quantized_matmul.py

```diff
@@ -97,9 +106,90 @@ def test_q_linear_hlo(self):
 
       output = torch.ops.xla.quantized_matmul(x, w_int, scaler)
       hlo = torch_xla._XLAC._get_xla_tensors_hlo([output])
-      print(hlo)
       self.assertTrue(re.search(r'bf16.*dot.*bf16.*s8', hlo) is not None)
 
+  def test_int4_pack_unpack(self):
+    x = torch.randint(-8, 7, (2, 12), dtype=torch.int8)
+    orig_dtypes = [
+        torch.int8,
+    ]
```

Comment:
does 4 bit pack/unpack only supported on int8 input?

Reply:
We don't go through pack and unpack flow right now, but i still keep it there in case it become useful some day, okay to remove